src/NextGen/examples/MParticleCommerceHandler.ts

```diff
@@ -0,0 +1,40 @@
+import { CommerceEvent } from '@mparticle/event-models';
+import { BaseMparticleDataHandlerStrategy } from '../dataHandler/dataHandlerStrategy';
+import { BaseMPMessage } from '../core/baseMPMessage';
+import { MParticleDataRepository } from '../dataRepository';
+import BatchManager from '../batchManager';
+import { DataHandlerType } from '../dataHandler/dataHandlerType';
+
+export default class MParticleCommerceHandler extends BaseMparticleDataHandlerStrategy<
+    CommerceEvent,
+    BaseMPMessage
+> {
+    private dataRepository: MParticleDataRepository;
+    constructor(
+        dataRepository: MParticleDataRepository,
+        batchManager: BatchManager
+    ) {
+        super(batchManager);
+        this.dataRepository = dataRepository;
+        this.batchManager = batchManager;
+    }
+
+    // QUESTION: Should we make immideateUpload optional with a default of false?
+    // QUESTION: Why should data be any rather than an actual commerce vent?
+    public saveData(data: any, immediateUpload: boolean): void {
```

Comment:
agreed on immediateUpload being option with default to false as really the only time it's true is for CommerceEvents I think.

Same comment as before re: any types, we should never use any